packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/renderer/palette/DiagramElementPalette.tsx

```diff
@@ -41,39 +40,36 @@ const useEdgePaletteStyle = makeStyles((theme) => ({
 
 const DiagramElementPaletteContent = ({ diagramElementId, labelId }: DiagramElementPaletteProps) => {
   const { setCurrentlyEditedLabelId } = useDiagramDirectEdit();
-  const { setReferencePosition, resetReferencePosition } = useLayout();
+  const { x: viewportX, y: viewportY, zoom: viewportZoom } = useViewport();
   const { x: paletteX, y: paletteY } = useDiagramElementPalette();
-  const { x, y } = usePaletteReferencePosition();
   const classes = useEdgePaletteStyle();
 
+  let x: number = 0;
+  let y: number = 0;
```

Comment:
It is difficult to see the difference between `x` and `paletteX` (same for `y` and `paletteY`).
I guess consider the scale, the other one does not.

Reply:
I've just moved this code but from my understanding paletteX / paletteY are absolute positions in the page, while x/y are positions within the viewport (i.e., 0x0 in the viewport can still be 400x600 in the page)